pyproject.toml

```diff
@@ -7,6 +7,48 @@ skip_magic_trailing_comma = true
 # for just these files, but doesn't seem possible yet.
 force-exclude = ".*_pb2.pyi"
 
+[tool.ruff]
```

Comment:
Moved this table above the `tool.ruff.isort` table, as I found the current order somewhat confusing